src/hotkey/hotkey_item.cpp

```diff
@@ -157,6 +157,7 @@ bool hotkey_base::matches(const SDL_Event &event) const
 void hotkey_base::save(config& item) const
 {
 	item["command"] = get_command();
+	item["disabled"] = is_disabled() ? config::attribute_value::s_true : config::attribute_value::s_false;
```

Comment:
This is not related to 'hot-key load/save functions' but rather to how config attributes work. Note that just writing `item["disabled"] = is_disabled()` is not only easier to read its also faster becasue with `config::attribute_value::s_true `the code would need to convert the string s_true to a bool, which is sure slower than just giving the bool in the first place.